src/NUnitFramework/tests/Internal/EventListenerTextWriterTests.cs

```diff
@@ -79,7 +81,7 @@ public void TestWriteStringArg1()
             ListenerWriter.Write(format, arg0);
             ListenerWriter.WriteLine(format, arg0);
 
-            var expected = "20 Apr 2017";
+            var expected = $"20 {CultureInfo.CurrentCulture.DateTimeFormat.GetAbbreviatedMonthName(4)} 2017";
```

Comment:
Why `CultureInfo.CurrentCulture` here and `CultureInfo.InvariantCulture` in LessThanConstraintTests?